packages/yourloops/test/integration/use-cases/patient-data-visualisation.ts

```diff
@@ -47,6 +52,13 @@ export const testDashboardDataVisualisation = async (patientDashboardLayoutParam
   await checkMonitoringAlertsCard()
 }
 
+export const testDashboardDataVisualisationForPatient = async (patientDashboardLayoutParams: PatientDashboardLayoutParams): Promise<void> => {
+  await checkPatientDashboardLayout(patientDashboardLayoutParams)
+  await checkPatientStatistics()
+  await checkDeviceUsageWidget()
+  expect(screen.queryByTestId('monitoring-alerts-card')).not.toBeInTheDocument()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Shouldn't this be done in `checkPatientDashboardLayout`?

Reply:
It's already done actually, I did not remember that 😄 The check is redundant, I'll remove it